seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -691,7 +691,9 @@
       ],
       "tests": [
         "assert(quotient === 2.2, 'message: The variable <code>quotient</code> should equal <code>2.2</code>.');",
-        "assert(/\\//.test(code), 'message: You should use the <code>/</code> operator');"
+        "assert(/4\\.40*\\s*\\/\\s*2\\.*0*/.test(code), 'message: You should use the <code>/</code> operator.');",
+        "assert(code.match(/quotient/g).length === 3, 'message: The quotient variable should only be assigned once.');"
```

Comment:
@erictleung Yeah, but this is caused by the code.match Regex tests now including the tail. So my only option are to remove the suggested solution from the json or remove the tail. I'm going to open an issue referencing this.